### PR TITLE
perl-graphviz: add graphviz dependency

### PR DESCRIPTION
**Pull request by teaguesterling**
Noticed that `perl-graphviz` did not build while testing #46213 due to a reported missing `dot` executable. Adding a `graphviz` dependency resolved this.

### CODE REVIEW

**Review by alalazo — APPROVED** [review]
Waiting for maintainers before merging

**Review by EbiArnie — APPROVED** [review]
OK

**Comment by EbiArnie** [comment]
Maybe this could also have some variants - GraphViz can have libgd for (e.g.) PNG, poppler for PDF etc.
Right now you would have to know and add the variants yourself to graphviz.
But that can come later, this is fine and an obvious improvement.

**Pull request merged**
